fulp_modules/Z_edits/race_bans/races.dm

```diff
@@ -1,15 +1,10 @@
-/datum/species/ethereal/spec_life(mob/living/carbon/human/H)
-	if(is_banned_from(H.ckey, RACE_ETHEREAL))
-		if(prob(50))
-			H.set_species(/datum/species/beefman)
-		else
-			H.set_species(/datum/species/human)
+/datum/species/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	if(is_banned_from(C.ckey, src.id))
```

Comment:
dont do src.var unless you have an arg named the same as a var on that object

Reply:
Noted for the future, was just being verbose. I'll fix it on my next go around